twake/backend/node/src/services/user/services/users/service.ts

```diff
@@ -80,15 +80,17 @@ export class UserService implements UsersServiceAPI {
     return this.repository.find(findFilter, findOptions);
   }
 
+  getByEmails(emails: string[]): Promise<User[]> {
+    return this.repository
+      .find({}, { $in: [["email_canonical", emails]] })
```

Comment:
Found it, the "IN" operator doesn't work with indexes. Could you fetch entities one by one instead of using a in ?